docs/en/observability/create-alerts.asciidoc

```diff
@@ -3,39 +3,72 @@
 
 [IMPORTANT]
 ====
-Ensure Alerting is already set up in {kib}. For details, see
+Make sure alerting is already set up in {kib}. For details, see
 {kibana-ref}/alerting-setup.html[Setup and prerequisites].
 ====
 
-Alerting enables you to detect complex *conditions* defined by a rule
+Alerting enables you to detect complex conditions defined by a *rule*
 within the Logs, Metrics, Uptime, and APM apps. When a condition is met, the rule
 tracks it as an *alert* and responds by triggering one or more *actions*.
 
-Listed on the <<view-observability-alerts,Alerts>> page are all of your
-observability alerts that have met a condition defined by a rule you created
-using the Logs, Metrics, Uptime, or APM apps.
+Alerts and rules related to {observability} apps including Logs, Metrics, Uptime, and APM can be managed in the {observability} UI.
+You can also manage {observability} app rules alongside rules for other apps from the {kibana-ref}/create-and-manage-rules.html[{kib} Management UI].
 
-IMPORTANT: Not all of the predefined rules in Stack Management will generate and 
-list an alert on the Alerts page. Only alerts generated by rules relating to Logs, Metrics, Uptime, and APM
-can be viewed on the Alerts page.
+[discrete]
+[[create-alerts-rules]]
+== Create rules
+
+The first step when setting up alerts is to create a rule.
+To create and manage rules related to {observability} apps,
+go to the {observability} *Alerts* page and click *Manage Rules* to navigate to the {observability} Rules page.
```

Comment:
So clicking *Manage Rules* here takes you to the **Rules page**?

Reply:
Yes clicking on the *Manage Rules* link in the *Alerts* page, takes you to the new Observability Rules page.